docs/src/administration/web/_index.md

```diff
@@ -112,19 +112,24 @@ to restore an existing backup.
 
 The URLs pull-down exposes the domains that can be used to access application environments from the web.
```

Comment:
```suggestion
The URLs drop-down exposes the domains that can be used to access application environments from the web.
```

Any other examples should in theory be changed, too. I just noticed this in passing. Not a big deal, since we have so much other inconsistent wording across the docs, but I can't not mention it. :speak_no_evil: